### DISCUSSION TITLE
Controlling fabricjs using x and y coordinates

### DISCUSSION THREAD

**Discussion by ashish140399** [fabricjs/fabric.js#8580] [Q&A]
I am making an app in which i pass x and y coordinate and can use the paint brush by just using x and y without using mouse events. Is there something which i can can use to make drawing using x and y coordinates.
Thanks

**Comment by LiHowe** [comment]
Maybe... you can use your point to invoke the brush `onMouseDown`,`onMouseMove` and `onMouseup` function with `forEach` or other loop function, use x and y coordinate as `new fabric.Point(x, y)` x and y

Here is a demo [Codepen](https://codepen.io/lihowe/pen/wvxJJZR)
